src/main/java/jenkins/plugins/office365connector/CardBuilder.java

```diff
@@ -104,8 +104,8 @@ private static String getCardThemeColor(Result result) {
         }
     }
 
-    private Section buildSection() {
-        String activityTitle = "Notification from " + getEscapedDisplayName();
+    private Section buildSection(String status) {
+        String activityTitle = "Notification from " + getEscapedDisplayName() + ": " + status;
```

Comment:
is this screen still valid https://github.com/jenkinsci/office-365-connector-plugin/blob/master/.README/teams.png ?

Reply:
Right, screenshot would be nice - will deploy and check